.travis.yml

```diff
@@ -0,0 +1,14 @@
+services:
+- docker
+
+env:
+  global:
+  - secure: "a/IAtWSXI+jUIQh27B5mNO7bxxCOB2gcB5R5+bQnGb8T+5ZrFxygLsjDiSU/RehD4HTBIfoNoHXkgPYRI6+YvZOnPFDKrYKkW/Wk4YmlGTx77+U9e0SZcMYL8fAK5zj5xrjN/kVO/rTkBuiOzxgKrbu0cTJiIEP7P3QZwPo87QGTy3prQT9yKxbYWSFmqrQgwe3BBrQCFgYIIOOBJUx1adyjw6B/tAzEaGF5hn0cV/V1eMVluSGFjhsQBVJZbjXhLg5SqMQVvQkGqZEYHha3li06oXiejGja+bPmgCfWnReZK3FFO6QanXyKvenNLqJCRRLcf7apLrPCYXYRDvJjLJWVQ1L3BGw6DmFZ4R3T71JjbwHrDeeutJzRyJ36xM5GdFjz1Jq+Xp7DYuCYhbRSBbImvgLbsU3UHytTukDDYlo/wRp5X9A4GWj4IJM+2TFSFJkQk+RgVyC+DHSCzsDJBtJW/mO+I9MpliE/0pb0QsvaA8llOuBtu00HzE1Q/vb8RAm0e7TxvsiKoxWjJ+/GqbrbqE4kxINsw/spqb/d8Hl/uSF/M31J7ZgtrzFvJmdytHJOA7minp4nwrw297diksmGZquBoo6hMCZNPfuF7lkP9cVA5AAccGy4wmkjU3a0P8Omwd6r+K3CH/YZUBIdZNL7tn8uavECertwSDNi+cw="
+  - secure: "SwGn9w7yO9yCyeJUfE8uaE5qbXYL/TqMExrIA1hHZs0Ie9+ww3sxo/KFJrpbl31caI8LturXbFNdylKjGlOo0viEHaw6JLg+A9WFa5RbjA1lp4LSoT9p7Sl/syrqOt2GFLsGM7Ya1AHbQWRYx8NwIcpvgcCK5d1xz+6DfPo3Gl0K6Xa1/UXww0aIHpbjbi4TK/JBbvKfP5dHG3Jv16H/a9CrV4hqwqxuy1C4t2N+43g5zTgO2npPbi2LAF4Vkf7lMIErYWlel+N+nI0X+MqI29d6MqGH0v7+8gkdRdnVaWK0Cb0BRveZ6xXLJT18NGNMnjKwompd5Y5kC6Xe8m2xHQbzgNncbE4NpC0y72j+APynWhf329euXwgAE1hHM9YUD6QiGsX7Bg8f7xGTmhAq18uZJidIndYAXQPYb2CSvxhOORB5ZIsleCPTqWC4fWtb/8Qm1xirawT+/vZLUDbRGchHSJp++nXEWOq9Cwf32gNh7yg5tgj73qKXQFBASHEA9iGfGk0GB1MseL6ba4rZTijGijaVbTvWWoq6XzC1F95c4uQFHd6Xrg09lJJ+PQWl+upXJT2XEVa9JHwdaArAM+vCOxruqmG4mh5LLLOvzjkr27XjqMryvyutwd8lhg5/LR1se8+RuZRzA9LIT8wBGPhbVV4cyhctIMbmOiSWCQg="
+  - secure: "RRdrXyE22napOrA30aXV03rDZXwDmxpxvuDZZY+sGO1gdrZmws0/tqT+IqVAOnLc17CUcSWY/wCEQBXGeU2NexFvQPrhUSNkO9v92/EuyJC/aUfaYv/IlGEsRk++TUUF8ipEu5QNL+IOnUD30GK4/7xYo5yY3hdXDWuKnyW2Lv4X5x/uaNTBECtTs37w0D/n7BPUCvLR/Vm4sOtB0TvO17yz2W5lkl1L8dNEuJ1W66aEeCUjqdJlQ/insU7e9uzy3AR5PCEyvlHPD4I/rYh+DRaWceiyf/+P22UIAKdJPdYSGn4hTNAxG0hqCyblkzmAQTTJBe+BNb3I43BXASO0bjFnH+YrQ554nddXLu6YQmt3zrqnpZ2hxVik1oBBOpSX/TKfhhuYNPXN0wiVKsMx+vVHgXNl1ayGWbgELwPVsg4riAZfaENbbiRLkfeCV9N8SstHfyk+670/25FEbQFJz9BtCuH/gx0hImVl/Xpkc8sbYRmv3oatQzE5O+uWxjTRZXkjY1T0WI/ynKtovD4tg+bn7B/IIYAjrujKdsLzja3UhWoQyPA/xufhEOH7sQUuPKzmLox4Rv0V1/APwBlffz1xLkdvoUhClYfOaugac/kZVOcl3whNH65wvQiGdf/US/Ny2uQmOxs1k+nMrsUd+uuj/uQ1bHngeS2qQ/E3lp0="
+
+script:
+- docker build --target test --tag todo-app:test .
+- docker run todo-app:test todo_app/tests/test_trello.py
+- docker run --env-file .env.test todo-app:test todo_app/tests/test_app.py
+- docker run --env TRELLO_BOARD_ID --env TRELLO_AUTH --env TRELLO_TOKEN todo-app:test todo_app/tests_e2e/test_e2e.py
```

Comment:
Shouldn't need TRELLO_BOARD_ID, because it should be creating its own temporary board.

Reply:
Would be slightly easier to maintain if you referenced the tests by folder so you only ever need two commands here `docker run --env-file .env.test todo-app:test todo_app/tests` and `docker run --env ... todo-app:test todo_app/tests_e2e`